src/components/clusters/wizards/rosa/ClusterRolesScreen/CustomerOIDCConfiguration.jsx

```diff
@@ -82,12 +82,14 @@ function CustomerOIDCConfiguration({
   const {
     getFieldProps,
     getFieldMeta,
-    values: { [FieldId.RegionalInstance]: regionalInstance },
+    values: { [FieldId.RegionalInstance]: regionalInstance, [FieldId.Hypershift]: hypershiftValue },
   } = useFormState();
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const [isRefreshLoading, setIsRefreshLoading] = useState(false);
   const [oidcConfigs, setOidcConfigs] = useState([]);
 
+  const isHypershift = hypershiftValue === 'true';
```

Comment:
please rename for clarity; update to `isHypershiftSelected`.  
(this avoids ambiguity IRT similar variables' semantics, like _'is an installed cluster with hypershift control-plane'_ or _'is "hypershift" feature-flag enabled'_).

same goes for other occurrences of this var' inside the wizards.